assets/constants.js

```diff
@@ -14,592 +13,593 @@
 
 var constants = {
   // Whether to allow custom event reporting to Google Analytics.
-  "CAN_SEND_ANALYTICS_EVENTS": false,
-
-  "ALL_CATEGORIES": ["Algebra", "Algorithms", "Architecture", "Arithmetic",
-    "Art", "Astronomy", "Biology", "Business", "Calculus", "Chemistry",
-    "Combinatorics", "Computing", "Economics", "Education", "Engineering",
-    "English", "Environment", "Gaulish", "Geography", "Geometry", "Government",
-    "Graph Theory", "History", "Languages", "Latin", "Law", "Logic",
-    "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
-    "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
-    "Statistics", "Trigonometry", "Welcome"],
-  "ACTIVITY_TYPE_EXPLORATION": "exploration",
-  "ACTIVITY_TYPE_COLLECTION": "collection",
-  "DISABLED_EXPLORATION_IDS": ["5"],
-  "TESTING_CONSTANT": "test",
-  "LIBRARY_TILE_WIDTH_PX": 208,
-  "DASHBOARD_TYPE_CREATOR": "creator",
-  "DASHBOARD_TYPE_LEARNER": "learner",
-  "DEFAULT_COLOR": "#a33f40",
-  "DEFAULT_THUMBNAIL_ICON": "Lightbulb",
-  "DEFAULT_CATEGORY_ICON": "Lightbulb",
+  CAN_SEND_ANALYTICS_EVENTS: false,
+
+  ALL_CATEGORIES: ['Algebra', 'Algorithms', 'Architecture', 'Arithmetic',
```

Comment:
Hi @sauravpratihar, please see line 4 of this file-- I think we need double quotes to interpret this as JSON?

Reply:
I think eslint conf file configured for single quote. [here](https://github.com/oppia/oppia/blob/develop/.eslintrc#L177)

Reply:
Ah -- I think we need to disable just that specific warning for all lines, because this file gets interpreted as JSON by the backend and JSON doesn't handle single quotes.

But the current rule is too broad, it disables everything. So we need to make it narrower to cover only what is needed.

Does that make sense? Sorry this wasn't clear earlier.

Reply:
so now we only need to solve this single quote issue by using **eslint-disable quotes** 
right?
also do we need double quotes to JSON key?

Reply:
Yes, double quotes are needed everywhere in JSON, I believe. I took a look at your other PR and left comments!